packages/datetime/src/datePickerCore.tsx

```diff
@@ -13,7 +13,8 @@ export interface IDatePickerLocaleUtils {
     formatWeekdayShort: (weekday: number, locale: string) => string;
     formatWeekdayLong: (weekday: number, locale: string) => string;
     getFirstDayOfWeek: (locale: string) => number;
-    getMonths: (locale: string) => string[];
+    // tslint:disable-next-line:max-line-length
+    getMonths: (locale: string) => [string, string, string, string, string, string, string, string, string, string, string, string];
```

Comment:
this is re-exported from ReactDayPicker itself; we cannot change the name. https://github.com/palantir/blueprint/pull/1014#discussion_r113601905 will actually allow me to remove this type entirely